metecho/api/migrations/0083_remove_all_null_strings.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.0.7 on 2020-06-08 21:36
+
+import sfdo_template_helpers.fields.string
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    atomic = False
```

Comment:
@wlonk Without these, I was getting an error:

```
django.db.utils.OperationalError: cannot ALTER TABLE "api_scratchorg" because it has pending trigger events
```